src/admin/orca-admin.ts

```diff
@@ -21,26 +21,26 @@ import {
   WhirlpoolClient,
   getWhirlpoolPda,
   NUM_REWARDS,
-  TickSpacing,
   getFeeTierPda,
 } from "@orca-so/whirlpool-client-sdk";
 import { resolveOrCreateATA } from "../utils/web3/ata-utils";
+import { PoolUtil } from "../utils/whirlpool/pool-util";
 
 export class OrcaAdmin {
   constructor(private readonly dal: OrcaDAL) {}
 
   public getInitPoolTx(param: InitPoolTxParam): { tx: TransactionBuilder; address: PublicKey } {
-    const { provider, initSqrtPrice, tokenMintA, tokenMintB, stable } = param;
+    const { provider, initSqrtPrice, tokenMintA, tokenMintB, tickSpacing } = param;
     const { programId, whirlpoolsConfig: whirlpoolConfigKey } = this.dal;
     const ctx = WhirlpoolContext.withProvider(provider, programId);
     const client = new WhirlpoolClient(ctx);
 
-    const tickSpacing = stable ? TickSpacing.Stable : TickSpacing.Standard;
+    const [_tokenMintA, _tokenMintB] = PoolUtil.orderMints(tokenMintA, tokenMintB);
```

Comment:
hmmm, i think this could be dangerous, since the initial price will be the inverse. the original behavior, where the transaction failed if the mints were the wrong order, was intentional. 

what if we instead check if the mints are in the correct order and throw an exception?